src/home/ExpandedCard.js

```diff
@@ -4,13 +4,34 @@ import {
   Text,
   ScrollView,
   Dimensions,
+  Image,
 } from 'react-native';
-import AddButton from './components/AddToLessonButton';
+import { useState, useEffect } from 'react';
+import { readFile } from '../services/routes/Local.js';
 import Header from '../Components/Header';
 
 const windowHeight = Dimensions.get('window').height;
 
-const ExpandedCard = ({ navigation }) => {
+const ExpandedCard = ({ route, navigation }) => {
+  const { cardPath } = route.params;
+  const [title, setTitle] = useState('');
+  const [cardImagePath, setCardImagePath] = useState(null);
+
+  useEffect(() => {
+    const readCardTitle = async () => {
+      try {
+        const cardTitlePath = cardPath + 'cardName.txt';
+        var cardNames = await readFile(cardTitlePath, 'utf-8');
+        cardNames = cardNames.substring(0, cardNames.length - 5);
+        setCardImagePath(cardPath + 'cardImage.jpg');
+        setTitle(cardNames);
+      } catch (err) {
+        console.log(err);
```

Comment:
```suggestion
        console.warn(err);
        setTitle('Could not load card preview. Please try again.');
```
Let's show something to the user in case something goes wrong.